core/src/main/java/org/jdbi/v3/mapper/CollectorFactory.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jdbi.v3.tweak;
+package org.jdbi.v3.mapper;
```

Comment:
Yeah, kinda.

I guess it's fine there. Although I don't mind packages with only a few classes, personally.